pythonnet/.gitignore

```diff
@@ -0,0 +1,3 @@
+mono/
+netfx/
+runtime/
```

Comment:
Wouldn't `runtime/` pattern generate a match for `/src/runtime/`?

Overall, I think all 3 are too generic.

Reply:
This directory is a build directory, it doesn't contain source code. The `.gitignore` file only applies to the directories within this one.